RELEASE-NOTES.md

```diff
@@ -1,5 +1,72 @@
+
+See the following PR for implementation details:
+
+- https://github.com/PrefectHQ/prefect/pull/11511
+
+
+## Toggle Deployment Schedule Status via `prefect.yaml`
+
+You can now toggle your deployment schedules between active and inactive in your prefect.yaml. This enables you to create deployments with initially inactive schedules, allowing for thorough testing or staged rollouts. 
+
+## Support for Python 3.12
+
+You can now install `prefect` using Python 3.12. This support is experimental and will be hardened in future releases.
+
+See the following PR for details:
+
+- https://github.com/PrefectHQ/prefect/pull/11306
+
+
+### Enhancements
+- Add an option through the CLI and python client to remove schedules from deployments — https://github.com/PrefectHQ/prefect/pull/11353
+- Add client methods to interact with global concurrency limit apis — https://github.com/PrefectHQ/prefect/pull/11415
+- Make name optional when saving a loaded block — https://github.com/PrefectHQ/prefect/pull/11592
+- Make subflows optional with `run_deployment` — https://github.com/PrefectHQ/prefect/pull/11611
+- Improve IDE support for `PrefectObjectRegistry.register_instances` decorated classes — https://github.com/PrefectHQ/prefect/pull/11617
+- Make the UI accessible via reverse proxy and add a `--no-install` flag to `prefect dev build-ui` — https://github.com/PrefectHQ/prefect/pull/11489
+- Improve UI build when using `prefect server start` - https://github.com/PrefectHQ/prefect/pull/11493
+- Improve error message when using `.deploy` — https://github.com/PrefectHQ/prefect/pull/11615
+
+### Fixes
+- Use default values (if any) if no run input is provided on resume — https://github.com/PrefectHQ/prefect/pull/11598
+- Prevent deployments with `RRule` schedules containing `COUNT` — https://github.com/PrefectHQ/prefect/pull/11600
+- Fix flows with class-based type hints based on `from __future__ import annotations` — https://github.com/PrefectHQ/prefect/pull/11578 & https://github.com/PrefectHQ/prefect/pull/11616
+- Raise `StepExecutionError` on non-zero `run_shell_script` return code during `prefect deploy` — https://github.com/PrefectHQ/prefect/pull/11604
+
+### Experimental
+- Enable flow runs to receive typed input from external sources — https://github.com/PrefectHQ/prefect/pull/11573
+
+### Documentation
+- Fix non-rendering link in Docker guide — https://github.com/PrefectHQ/prefect/pull/11574
+- Update deployment and flow concept docs — https://github.com/PrefectHQ/prefect/pull/11576
+- Add examples for custom triggers for automations — https://github.com/PrefectHQ/prefect/pull/11589
+- Add send/receive documentation to `run_input` module docstring — https://github.com/PrefectHQ/prefect/pull/11591
+- Add Automations Guide — https://github.com/PrefectHQ/prefect/pull/10559
+- Fix storage guide links and reference — https://github.com/PrefectHQ/prefect/pull/11602
+- Fix typo in deploy guide — https://github.com/PrefectHQ/prefect/pull/11606
```

Comment:
```suggestion
- Fix typo in `prefect deploy` guide — https://github.com/PrefectHQ/prefect/pull/11606
```

i'm only guessing but I think this is what you're referring to? or `.deploy`? or both?